esx_service/cli/vmdkops_admin.py

```diff
@@ -916,10 +916,7 @@ def generate_tenant_ls_rows(tenant_list):
         uuid = tenant.id
         name = tenant.name
         description = tenant.description
-        if not tenant.default_datastore_url or tenant.default_datastore_url == auth_data_const.DEFAULT_DS_URL:
-            default_datastore = ""
-        else:
-            default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
+        default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
```

Comment:
a. Does this work for the xESX scenario? Unless the datastore is shared, get_datastore_name() won't return a valid name.

b. If the tenant.default_datastore_url is VM_DS, then it should map to the datastore name for each VM in the tenant. Which need not be the same for every VM in the tenant. Although, to keep it simple, we may want to restrict VMs in a tenant to include only those that are on the same DS.